src/main/java/com/books/dao/concrete/SQL/PublisherSQLDAO.java

```diff
@@ -45,26 +45,30 @@ public Publisher remove(int id) {
     @Override
     public List<Publisher> getList() {
         String getAllPublishers = String.format("select * from %s ", PUBLISHER_TABLE_NAME);
-
-        return jdbcTemplate.query(getAllPublishers, (rs, rn) -> {
+        List<Publisher> list;
+        list = jdbcTemplate.query(getAllPublishers, (rs, rn) -> {
             Publisher publisher = new Publisher();
```

Comment:
Here you break DRY principle. Why not bear this RowMapper to a separate class and reuse?